.redhat-ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# F25 currently has 2.2.1, so install from pypi
+pip install ansible==2.2.2.0
+
+# do a simple ping to make sure the nodes are available
+ansible -vvv -i .redhat-ci.inventory nodes -a 'rpm-ostree status'
```

Comment:
If we just want to know the hosts are available, why not use the simpler [`ping`](http://docs.ansible.com/ansible/ping_module.html) command?